sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/autorest.md

```diff
@@ -218,6 +220,27 @@ rename-mapping:
   SyncType: BackupInstanceSyncType
   ScheduleBasedBackupCriteria.daysOfTheWeek: DaysOfWeek
   ScheduleBasedBackupCriteria.weeksOfTheMonth: WeeksOfMonth
+  DeletedBackupInstanceResource: DataProtectionDeletedBackupInstance
```

Comment:
Is `DeletedDataProtection`BackupInstance` better?

Reply:
Good point, fixed!